google/cloud/bigquery/dataset.py

```diff
@@ -527,13 +527,46 @@ class Dataset(object):
         "friendly_name": "friendlyName",
         "default_encryption_configuration": "defaultEncryptionConfiguration",
         "storage_billing_model": "storageBillingModel",
+        "default_rounding_mode": "defaultRoundingMode",
     }
 
     def __init__(self, dataset_ref) -> None:
         if isinstance(dataset_ref, str):
             dataset_ref = DatasetReference.from_string(dataset_ref)
         self._properties = {"datasetReference": dataset_ref.to_api_repr(), "labels": {}}
 
+    @property
+    def default_rounding_mode(self):
+        """
+        optional[str]: it could have one of the following possible value.
+        ROUNDING_MODE_UNSPECIFIED: Unspecified will default to using ROUND_HALF_AWAY_FROM_ZERO.
+        ROUND_HALF_AWAY_FROM_ZERO: ROUND_HALF_AWAY_FROM_ZERO rounds half values away from zero when applying precision
+                                    and scale upon writing of NUMERIC and BIGNUMERIC values.
+                                    For Scale: 0 1.1, 1.2, 1.3, 1.4 => 1 1.5, 1.6, 1.7, 1.8, 1.9 => 2
+        ROUND_HALF_EVEN: ROUND_HALF_EVEN rounds half values to the nearest even value when applying precision and scale
+                        upon writing of NUMERIC and BIGNUMERIC values.
+                        For Scale: 0 1.1, 1.2, 1.3, 1.4 => 1 1.5 => 2 1.6, 1.7, 1.8, 1.9 => 2 2.5 => 2
+        """
```

Comment:
Thank you for adding the docstring. I think its format is not passing the docs test, see [testing logs](https://source.cloud.google.com/results/invocations/6b4f735e-7003-4da4-85fa-567231fcaa58/targets/cloud-devrel%2Fclient-libraries%2Fpython%2Fgoogleapis%2Fpython-bigquery%2Fpresubmit%2Fpresubmit/log) for more detailed information. [Here is an example](https://github.com/googleapis/python-bigquery/blob/8f187e62b256882c31b84c1d2373d56ad28cabb4/google/cloud/bigquery/dataset.py#L767-L784) for docstring format.

Reply:
@Linchin  would a linter catch this ?

Reply:
done.  i fixed it and run `nox -s docs` without any error.